scripts/create-distributable.ts

```diff
@@ -0,0 +1,67 @@
+import archiver from 'archiver';
+import fs from 'fs';
+import path from 'path';
+import colorLog from './log';
+
+async function createSafariDistributable() {
+	colorLog('TODO: Make safari distributable work', 'error');
+}
+
+async function createSrcArchive() {
+	const curDir = path.resolve(path.dirname(''));
+	const outputFile = `${curDir}/web-scrobbler-src.zip`;
+
+	const output = fs.createWriteStream(outputFile);
+
+	const archive = archiver('zip', {
+		zlib: { level: 0 },
+	});
+
+	archive.pipe(output);
+
+	// TODO: make a glob that gets the top level files and doesnt nuke the system
```

Comment:
What was the issue here?

Reply:
I think it went recursive and made a huge archive that nearly crashed my computer

Reply:
Fixed it now that I had some fresh eyes